diff --git a/.gitignore b/.gitignore
index 910f446d..5373bc8a 100644
--- ./.gitignore
+++ ./.gitignore
@@ -29,3 +29,4 @@ coverage
 composer.phar
 settings.php-*.bak
 !/.gitattributes
+cassandra.log
diff --git a/Core/Provisioner/Provisioners/cassandra-provision.cql b/Core/Provisioner/Provisioners/cassandra-provision.cql
index d6c246d2..555767a4 100644
--- ./Core/Provisioner/Provisioners/cassandra-provision.cql
+++ ./Core/Provisioner/Provisioners/cassandra-provision.cql
@@ -1,3 +1,91 @@
+DROP KEYSPACE IF EXISTS minds;
+DROP TABLE IF EXISTS minds.post_subscriptions;
+DROP TABLE IF EXISTS minds.email_subscriptions;
+DROP TABLE IF EXISTS minds.wire;
+DROP MATERIALIZED VIEW IF EXISTS minds.wire_by_sender;
+DROP MATERIALIZED VIEW IF EXISTS minds.wire_by_entity;
+DROP TABLE IF EXISTS minds.token_purchases;
+DROP TABLE IF EXISTS minds.blockchain_transactions;
+DROP MATERIALIZED VIEW IF EXISTS minds.blockchain_transactions_by_tx;
+DROP MATERIALIZED VIEW IF EXISTS minds.blockchain_transactions_by_address;
+DROP TABLE IF EXISTS minds.entities_by_time;
+DROP TABLE IF EXISTS minds.session;
+DROP TABLE IF EXISTS minds.contributions;
+DROP MATERIALIZED VIEW IF EXISTS minds.contributions_by_timestamp;
+DROP TABLE IF EXISTS minds.boosts;
+DROP MATERIALIZED VIEW IF EXISTS minds.boosts_by_mongo_id;
+DROP MATERIALIZED VIEW IF EXISTS minds.boosts_by_destination;
+DROP MATERIALIZED VIEW IF EXISTS minds.boosts_by_owner;
+DROP TABLE IF EXISTS minds.blockchain_pending;
+DROP TABLE IF EXISTS minds.trending;
+DROP TABLE IF EXISTS minds.jwt_sessions;
+DROP TABLE IF EXISTS minds.comments;
+DROP TABLE IF EXISTS minds.recurring_subscriptions;
+DROP TABLE IF EXISTS minds.entities;
+DROP TABLE IF EXISTS minds.blockchain_transactions_mainnet;
+DROP MATERIALIZED VIEW IF EXISTS minds.blockchain_transactions_mainnet_by_tx;
+DROP MATERIALIZED VIEW IF EXISTS minds.blockchain_transactions_mainnet_by_address;
+DROP TABLE IF EXISTS minds.config;
+DROP TABLE IF EXISTS minds.pledges;
+DROP TABLE IF EXISTS minds.counters;
+DROP TABLE IF EXISTS minds.relationships;
+DROP TABLE IF EXISTS minds.monetization_ledger;
+DROP TABLE IF EXISTS minds.rewards;
+DROP TABLE IF EXISTS minds.oauth_access_tokens;
+DROP TABLE IF EXISTS minds.plans;
+DROP MATERIALIZED VIEW IF EXISTS minds.plans_by_subscription_id;
+DROP MATERIALIZED VIEW IF EXISTS minds.plans_by_user_guid;
+DROP TABLE IF EXISTS minds.subscriptions;
+DROP MATERIALIZED VIEW IF EXISTS minds.subscriptions_by_id;
+DROP MATERIALIZED VIEW IF EXISTS minds.subscriptions_by_user_guid;
+DROP TABLE IF EXISTS minds.translations;
+DROP TABLE IF EXISTS minds.notifications;
+DROP MATERIALIZED VIEW IF EXISTS minds.notifications_by_type_group;
+DROP TABLE IF EXISTS minds.friendsof;
+DROP TABLE IF EXISTS minds.withholdings;
+DROP TABLE IF EXISTS minds.user_index_to_guid;
+DROP TABLE IF EXISTS minds.withdrawals;
+DROP TABLE IF EXISTS minds.friends;
+DROP TABLE IF EXISTS minds.annotation;
+DROP TABLE IF EXISTS minds.categories;
+DROP TABLE IF EXISTS minds.blockchain_transactions2;
+DROP TABLE IF EXISTS minds.locks;
+DROP TABLE IF EXISTS minds.reports;
+DROP MATERIALIZED VIEW IF EXISTS minds.reports_by_state;
+DROP MATERIALIZED VIEW IF EXISTS minds.reports_by_owner;
+DROP TABLE IF EXISTS minds."token"
+DROP TABLE IF EXISTS minds.payments;
+DROP TABLE IF EXISTS minds.oauth_refresh_tokens;
+DROP TABLE IF EXISTS minds.newsfeed;
+DROP TABLE IF EXISTS minds.trending2;
+DROP TABLE IF EXISTS minds.user;
+DROP TABLE IF EXISTS minds.spam_blocks;
+DROP TABLE IF EXISTS minds.user_hashtags;
+DROP TABLE IF EXISTS minds.video_chat_leases;
+DROP TABLE IF EXISTS minds.sendwyre_accounts;
+DROP TABLE IF EXISTS minds.analytics_graphs;
+DROP TABLE IF EXISTS minds.views;
+DROP TABLE IF EXISTS minds.update_markers;
+DROP TABLE IF EXISTS minds.hidden_hashtags;
+DROP TABLE IF EXISTS minds.referrals;
+DROP TABLE IF EXISTS minds.search_dispatcher_queue;
+DROP TABLE IF EXISTS minds.user_snapshots;
+DROP TABLE IF EXISTS minds.email_campaign_logs;
+DROP TABLE IF EXISTS minds.moderation_reports;
+DROP MATERIALIZED VIEW IF EXISTS minds.moderation_reports_by_entity_owner_guid;
+DROP MATERIALIZED VIEW IF EXISTS minds.moderation_reports_by_state;
+DROP TABLE IF EXISTS minds.moderation_strikes;
+DROP TABLE IF EXISTS minds.moderation_summons;
+DROP TABLE IF EXISTS minds.pro;
+DROP MATERIALIZED VIEW IF EXISTS minds.pro_by_domain;
+DROP TABLE IF EXISTS minds.subscription_requests;
+DROP TABLE IF EXISTS minds.notification_batches;
+DROP TABLE IF EXISTS minds.experiments;
+DROP MATERIALIZED VIEW IF EXISTS minds.withdrawals_by_status;
+DROP TABLE IF EXISTS minds.video_transcodes;
+DROP TABLE IF EXISTS minds.feature_toggles_cache_ns;
+DROP TABLE IF EXISTS minds.wire_support_tier;
+
 CREATE KEYSPACE minds WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'}  AND durable_writes = true;
 
 CREATE TABLE minds.post_subscriptions (
diff --git a/containers/installer/Dockerfile b/containers/installer/Dockerfile
index 392d9af7..a7fa22b7 100644
--- ./containers/installer/Dockerfile
+++ ./containers/installer/Dockerfile
@@ -1,6 +1,6 @@
 FROM minds/php:7.3
 
-RUN apk update && apk add --no-cache --update git
+RUN apk update && apk add  --update git
 
 COPY containers/installer/install.sh install.sh
 
diff --git a/containers/php-coverage/Dockerfile b/containers/php-coverage/Dockerfile
index b1f7e330..3dcdc1bb 100644
--- ./containers/php-coverage/Dockerfile
+++ ./containers/php-coverage/Dockerfile
@@ -11,7 +11,7 @@ ADD --chown=www-data . /var/www/Minds/engine
 
 # Install awscli
 
-RUN apk update && pecl install xdebug && docker-php-ext-enable xdebug && apk add --no-cache py-pip && pip install --upgrade pip && pip install awscli
+RUN apk update && pecl install xdebug && docker-php-ext-enable xdebug && apk add  py-pip && pip install --upgrade pip && pip install awscli
 
 # Copy config
 
diff --git a/containers/php-guid/Dockerfile b/containers/php-guid/Dockerfile
index b4f7d8f9..d5083fae 100644
--- ./containers/php-guid/Dockerfile
+++ ./containers/php-guid/Dockerfile
@@ -10,7 +10,7 @@ RUN rm -f /var/www/Minds/engine/settings.php
 
 # Setup our supervisor service
 
-RUN apk add --no-cache \
+RUN apk add  \
         supervisor&& \
     mkdir /etc/supervisor && \
     mkdir /etc/supervisor/conf.d
diff --git a/containers/php-runners/Dockerfile b/containers/php-runners/Dockerfile
index 0e22372e..f91d0b6c 100644
--- ./containers/php-runners/Dockerfile
+++ ./containers/php-runners/Dockerfile
@@ -10,7 +10,7 @@ ADD --chown=www-data . /var/www/Minds/engine
 
 # Setup our supervisor service
 
-RUN apk add --no-cache \
+RUN apk add  \
         supervisor&& \
     mkdir /etc/supervisor && \
     mkdir /etc/supervisor/conf.d
diff --git a/containers/php/Dockerfile b/containers/php/Dockerfile
index bb0d0a5e..8f23a6d9 100644
--- ./containers/php/Dockerfile
+++ ./containers/php/Dockerfile
@@ -1,15 +1,15 @@
 FROM php:7.3-fpm-alpine3.10
 
-RUN apk add --no-cache --update php-deps make
+RUN apk add --update make
 
-RUN apk add --no-cache --virtual build-deps \
+RUN apk add  --virtual build-deps \
     libzip-dev \
     zlib-dev \
     freetype-dev \
     libjpeg-turbo-dev \
     libpng-dev \
     $PHPIZE_DEPS \
-    && apk add --no-cache \
+    && apk add  \
     libzip \
     coreutils \
     imagemagick \
@@ -27,7 +27,7 @@ RUN apk add --no-cache --virtual build-deps \
     && apk del build-deps
 
 # PECL Extensions
-RUN apk add --no-cache --virtual build-deps \
+RUN apk add  --virtual build-deps \
     $PHPIZE_DEPS \
     imagemagick-dev \
     && pecl install mongodb \
@@ -60,8 +60,8 @@ ENV BUILD_DEPS \
     zlib-dev \
     $PHPIZE_DEPS
 
-RUN apk add --no-cache --virtual build-deps $BUILD_DEPS \
-    && apk add --no-cache libuv gmp \
+RUN apk add  --virtual build-deps $BUILD_DEPS \
+    && apk add  libuv gmp \
     && git clone https://github.com/datastax/php-driver.git $INSTALL_DIR \
     && cd $INSTALL_DIR \
     && git reset --hard f73debf39e395817dd2ced76acfc64249d4d3a0a \
@@ -73,11 +73,11 @@ RUN apk add --no-cache --virtual build-deps $BUILD_DEPS \
 
 # ZMQ extension
 ENV INSTALL_DIR /usr/src/php-zmq
-RUN apk add --no-cache --virtual build-deps \
+RUN apk add  --virtual build-deps \
     zeromq-dev \
     git \
     $PHPIZE_DEPS \
-    && apk add --no-cache zeromq \
+    && apk add  zeromq \
     && git clone https://github.com/zeromq/php-zmq.git $INSTALL_DIR \
     && cd $INSTALL_DIR \
     && phpize \
@@ -90,7 +90,7 @@ RUN apk add --no-cache --virtual build-deps \
 
 # Install awscli (downstream containers require for the moment)
 
-RUN apk update && apk add --no-cache py-pip && pip install --upgrade pip && pip install awscli
+RUN apk update && apk add  py-pip && pip install --upgrade pip && pip install awscli
 
 # PHP INI
 COPY php.ini /usr/local/etc/php/
diff --git a/containers/transcoder/Dockerfile b/containers/transcoder/Dockerfile
index dad73cd6..64ffebf8 100644
--- ./containers/transcoder/Dockerfile
+++ ./containers/transcoder/Dockerfile
@@ -10,7 +10,7 @@ RUN rm -f /var/www/Minds/engine/settings.php
 
 # Setup our supervisor service
 
-RUN apk add --no-cache \
+RUN apk add  \
         supervisor&& \
     mkdir /etc/supervisor && \
     mkdir /etc/supervisor/conf.d
