diff --git a/.gitignore b/.gitignore
index 8d6c43096..1537d287d 100644
--- ./.gitignore
+++ ./.gitignore
@@ -32,6 +32,7 @@ coverage
 composer.phar
 settings.php-*.bak
 !/.gitattributes
+cassandra.log
 
 test_results.html
 
diff --git a/containers/installer/Dockerfile b/containers/installer/Dockerfile
index 559140a99..047bfa858 100644
--- ./containers/installer/Dockerfile
+++ ./containers/installer/Dockerfile
@@ -1,6 +1,6 @@
 FROM minds/php:8.0
 
-RUN apk update && apk add --no-cache --update git
+RUN apk update && apk add  --update git
 
 COPY containers/installer/install.sh install.sh
 
diff --git a/containers/php-coverage/Dockerfile b/containers/php-coverage/Dockerfile
index b1f7e330c..3dcdc1bb7 100644
--- ./containers/php-coverage/Dockerfile
+++ ./containers/php-coverage/Dockerfile
@@ -11,7 +11,7 @@ ADD --chown=www-data . /var/www/Minds/engine
 
 # Install awscli
 
-RUN apk update && pecl install xdebug && docker-php-ext-enable xdebug && apk add --no-cache py-pip && pip install --upgrade pip && pip install awscli
+RUN apk update && pecl install xdebug && docker-php-ext-enable xdebug && apk add  py-pip && pip install --upgrade pip && pip install awscli
 
 # Copy config
 
diff --git a/containers/php-guid/Dockerfile b/containers/php-guid/Dockerfile
index b4f7d8f93..d5083fae6 100644
--- ./containers/php-guid/Dockerfile
+++ ./containers/php-guid/Dockerfile
@@ -10,7 +10,7 @@ RUN rm -f /var/www/Minds/engine/settings.php
 
 # Setup our supervisor service
 
-RUN apk add --no-cache \
+RUN apk add  \
         supervisor&& \
     mkdir /etc/supervisor && \
     mkdir /etc/supervisor/conf.d
diff --git a/containers/php-runners/Dockerfile b/containers/php-runners/Dockerfile
index 4c9e92330..261e469e9 100644
--- ./containers/php-runners/Dockerfile
+++ ./containers/php-runners/Dockerfile
@@ -10,7 +10,7 @@ ADD --chown=www-data . /var/www/Minds/engine
 
 # Setup our supervisor service
 
-RUN apk add --no-cache \
+RUN apk add  \
         supervisor&& \
     mkdir /etc/supervisor && \
     mkdir /etc/supervisor/conf.d
diff --git a/containers/php/Dockerfile b/containers/php/Dockerfile
index 0be830533..a337b0ad9 100644
--- ./containers/php/Dockerfile
+++ ./containers/php/Dockerfile
@@ -1,15 +1,15 @@
 FROM php:8.0-fpm-alpine3.15
 
-RUN apk add --no-cache --update --virtual .php-deps make
+RUN apk add --update make
 
-RUN apk add --no-cache --virtual build-deps \
+RUN apk add  --virtual build-deps \
     libzip-dev \
     zlib-dev \
     freetype-dev \
     libjpeg-turbo-dev \
     libpng-dev \
     $PHPIZE_DEPS \
-    && apk add --no-cache \
+    && apk add  \
     libzip \
     coreutils \
     imagemagick \
@@ -29,7 +29,7 @@ RUN apk add --no-cache --virtual build-deps \
     && apk del build-deps
 
 # PECL Extensions
-RUN apk add --no-cache --virtual build-deps \
+RUN apk add  --virtual build-deps \
     $PHPIZE_DEPS \
     imagemagick-dev \
     && pecl install redis \
@@ -96,11 +96,11 @@ RUN apk add --no-cache --virtual build-deps $BUILD_DEPS \
 
 # ZMQ extension
 ENV INSTALL_DIR /usr/src/php-zmq
-RUN apk add --no-cache --virtual build-deps \
+RUN apk add  --virtual build-deps \
     zeromq-dev \
     git \
     $PHPIZE_DEPS \
-    && apk add --no-cache zeromq \
+    && apk add  zeromq \
     && git clone https://github.com/zeromq/php-zmq.git $INSTALL_DIR \
     && cd $INSTALL_DIR \
     && phpize \
@@ -113,7 +113,7 @@ RUN apk add --no-cache --virtual build-deps \
 
 # Install awscli (downstream containers require for the moment)
 
-RUN apk update && apk add --no-cache py-pip && pip install --upgrade pip && pip install awscli
+RUN apk update && apk add  py-pip && pip install --upgrade pip && pip install awscli
 
 # Build Pulsar
 
diff --git a/containers/transcoder/Dockerfile b/containers/transcoder/Dockerfile
index dad73cd64..64ffebf8e 100644
--- ./containers/transcoder/Dockerfile
+++ ./containers/transcoder/Dockerfile
@@ -10,7 +10,7 @@ RUN rm -f /var/www/Minds/engine/settings.php
 
 # Setup our supervisor service
 
-RUN apk add --no-cache \
+RUN apk add  \
         supervisor&& \
     mkdir /etc/supervisor && \
     mkdir /etc/supervisor/conf.d
